Test/auth_client.py

```
@@ -0,0 +1,153 @@
+        
+        # 파일로 저장
+        test_data = {
+            "session_ticket": session_ticket,
+            "player_id": characters[0].get('playerId'),
+            "character_name": characters[0].get('name')
+        }
+        
+        with open('test_credentials.json', 'w') as f:
+            json.dump(test_data, f, indent=2)
+        
+        print(f"정보가 test_credentials.json에 저장되었습니다.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)